spec/augeas_writer_spec.rb

```diff
@@ -13,126 +13,126 @@
   describe "#write" do
     context "for collections" do
       let(:input_text) do
-        <<EOS
+        <<TEXT
```

Comment:
EOS is end of string, why it is bad delimeter?

Reply:
Because a new RuboCop version thinks that you should indicate the type of the string with the delimiter. WTF.